lib/tasks/server/middleware/tests-server/index.js

```diff
@@ -41,7 +41,8 @@ TestsServerAddon.prototype.serverMiddleware = function(config) {
         var filePath = path.join(results.directory, assetPath);
 
         if (!existsSync(filePath) || !fs.statSync(filePath).isFile()) {
-          var newURL = baseURL + '/tests/index.html';
+          // N.B., `baseURL` will end with a slash as it went through `cleanBaseURL`
+          var newURL = baseURL + 'tests/index.html';
```

Comment:
This appeared to be an existing bug. Without this change I got a double `/`, making my assertions fail: `/braden/+//tests/index.html`.